Makefile

```diff
@@ -28,6 +33,8 @@ build-plugin: setup-plugin
 
 build: build-core build-plugin
 
+build-local: setup-local build-core build-plugin
```

Comment:
With this, if you are building using a local go-car-mirror, you can run `make build-local`.